js/webauth.js

```diff
@@ -191,6 +200,7 @@ module.exports = function(authManager) {
       const handlers = getRelevantHandlers(authManager, req.body);
       for (const handler of handlers) {
         const pluginID = handler.pluginID;
+        authLogger.debug(`User logout for auth handler ${pluginID}`);
```

Comment:
Which user? For a given handler, there should be 1 user that is associated with a session. Since we know the handler, don't we know the user?

Reply:
I think that "user" is a handler-level concept.  My idea was that we print the handler's result here: https://github.com/zowe/zlux-proxy-server/blob/cb8183d6a70127b431c4b0d5292cf11a9f53922c/js/webauth.js#L181 and currently it's just `{ success: true }`.  We could make the handler add there any information it wants to report, like the user name, if applicable.